docs/software/devtools/debug/linaro.md

```diff
@@ -0,0 +1,294 @@
+`/user-tools/activate` script will make the forge executables available in your
+environment path.
+
+[//]: <> (The `/user-tools/activate` script will make the forge executables available)
+[//]: <> (in your environment, and **must be run before** any other uenv view command)
+[//]: <> (otherwise the environment variables set from the other uenv will be)
+[//]: <> (overwritten.)
+
+## User guide
+
+In order to debug your code on Alps, you need to:
+
+1. pull the linaro-forge uenv on one of the target Alps vCluster
+- install the Linaro Forge client on your local system (desktop/laptop)
+- build an executable with debug flags
+- launch a job with the debugger on Alps
+- start debugging/profiling
+
+### Pull the linaro-forge uenv on Alps
+
+The first step is to download the latest version of linaro-forge that is
+available on the system(s). First, SSH into the target system, then use the
+`uenv image find` command to list the available versions on the system:
+
+```
+> uenv image find linaro-forge
+uenv                    arch   system  id                size(MB)  date
+linaro-forge/24.1.1:v1  gh200  daint   e0e79f5c3e6a8ee0  365       2025-02-12
+```
+
+In this example, there is a single version available. Next we pull the image so
+that it is available locally.
+
+```
+> uenv image pull linaro-forge/24.1.1:v1
+pulling e0e79f5c3e6a8ee0 100.00% --- 365/365 (0.00 MB/s)
+```
+
+It will take a few seconds to download the image. Once complete, check that it
+was downloaded using the `uenv image ls` command:
+
+```
+> uenv image ls linaro-forge
+uenv                            arch   system  id                size(MB)  date
+linaro-forge/24.1.1:v1          gh200  daint   e0e79f5c3e6a8ee0     365    2025-01-22
+```
+
+### Install and configure the Linaro client on your local machine
+
+We recommend installing the [desktop client](https://www.linaroforge.com/downloadForge) 
+on your local workstation/laptop. It can be downloaded for a selection of
+operating systems. The client can be configured to connect with the debug jobs
+running on Alps, offering a better user experience compared to running with X11
+forwarding. Once installed, the client needs to be configured to connect to
+the vCluster on which you are working.
+
+First, start the client on your laptop:
+
+=== "Linux"
+
+    The path will change if you have installed a different version, or if it
+    has been installed in a non-standard installation location.
+
+    ```bash
+    $HOME/linaro/forge/24.1.1/bin/ddt
+    ```
+
+=== "MacOS"
+
+    The path will change if you have installed a different version, or if it
+    has been installed in a non-standard installation location.
+
+    ```bash
+    open /Applications/Linaro\ Forge\ Client\ 24.1.1.app/
+    ```
+
+Next, configure a connection to the target system.
+Open the *Remote Launch* menu and click on *configure* then *Add*. 
+Examples of the settings are below.
+
+=== "Eiger"
```

Comment:
Maybe it would be better to show `Daint` or `Clariden` by default, which should have the largest user base?